md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
```

Comment:
I find this doc very interesting. 
I'd say that slack etiquette vs. git commit formats are two different deals, and would probably split out the part about git commit into the workflow doc for instance.